### PR TITLE
Update FAQ section

### PR DESCRIPTION
**Pull request by YamenMerhi**

### CODE REVIEW

File: docs/faq/lukso-standard-proposal.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+# Frequently Asked Questions
+
+## What are LUKSO Standard Proposals ?
+
+LUKSO Standard Proposals, often abbreviated as **LSPs**, serve as building blocks designed to create a wide range of features and possibilities within the LUKSO ecosystem and other similar blockchain networks. With these LSPs, the mission is to have standards that facilitate the use of blockchain tools that abstract the blockchain complexity, effectively minimizing the technical problems users may encounter. By abstracting this complexity, LSPs make blockchain technology as user-friendly and accessible as the familiar Web 2.0 technologies. This ease of use paves the way for broader acceptance and integration of blockchain systems in our everyday digital interactions.
+
+## Why are LSPs not proposed as ERCs?
+
+The LUKSO Standard Proposals (LSPs) are part of a larger set of core standards that are all interoperable and cohesive together. Nonetheless, they do function equally for any EVM chain, the ability to make these standards understood requires a less noisy and neutral place, than the current ERC space.
```
Comment (CJ42) [inline docs/faq/lukso-standard-proposal.md:15]:
```suggestion
The LUKSO Standard Proposals (LSPs) are part of a larger set of core standards that are all interoperable and cohesive together. Nonetheless, they do function equally for any EVM chain, the ability to make these standards understood requires a less noisy and neutral place than the current ERC space.
```

File: docs/faq/lukso-standard-proposal.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Therefore, we deem a separate repo **[(LIPs/LSPs)](https://github.com/lukso-network/LIPs)** as necessary for the success and transparency of these new sets of standards, many of which use ERCs (ERC165, ERC173, ERC725) as a core basis.
```
Comment (CJ42) [inline docs/faq/lukso-standard-proposal.md:17]:
```suggestion
Therefore, we consider a separate repo **[(LIPs/LSPs)](https://github.com/lukso-network/LIPs)** to be necessary for the success and transparency of these new sets of standards, many of which use ERCs (ERC165, ERC173, ERC725) as a core basis.
```

File: docs/faq/lukso-standard-proposal.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+## What is a transaction Relayer?
+
+A transaction relayer service, often just referred to as a "relayer", is a solution designed to improve the blockchain user experience by addressing the issue of gas fees - the costs users pay when executing transactions on a blockchain network.
+
+Normally, users need to possess the blockchain's native coin to pay these gas fees. However, a relayer works around this problem. When a user want to execute a transaction, he could sign a message from their account and send it to the relayer, the relayer then will have the permission and ability to execute transactions on their behalf. Essentially, the relayer pays the gas fees for the user.
```
Comment (CJ42) [inline docs/faq/lukso-standard-proposal.md:23]: not sure if the verb "possess" can be used in this sentence. It sounds a bit unusual to me.

Maybe @CallumGrindle can give some insights here? :)

File: docs/faq/lukso-standard-proposal.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+Payment to the relayer can be handled in different ways. One option is an on-chain payment where the fees are deducted directly from a balance within the blockchain. Alternatively, users can set up an off-chain agreement with the relayer service, establishing a monthly allowance for relayed transactions. This flexibility allows users to choose the method that best suits their needs.
```
Comment (CJ42) [inline docs/faq/lukso-standard-proposal.md:25]: suggested changes:

```suggestion
Payment to the relayer can be handled in different ways. One option is an on-chain payment where the fees are deducted directly from a balance within the blockchain. Alternatively, users can set up an off-chain agreement with the relayer service, via a subscription that give them a monthly allowance for relayed transactions. This flexibility allows users to choose the method that best suits their needs.
```

File: docs/faq/lukso-standard-proposal.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+## What is a Social Recovery?
+
+For conventional Ethereum accounts, also known as Externally Owned Accounts (EOAs), losing your private key can be disastrous, as it's the sole means of accessing your account. If it's lost, access to the account and all its associated assets are lost as well.
```
Comment (CJ42) [inline docs/faq/lukso-standard-proposal.md:29]: typo:

```suggestion
For conventional Ethereum accounts, also known as Externally Owned Accounts (EOAs), loosing your private key can be disastrous, as it's the sole means of accessing your account. If it's lost, access to the account and all its associated assets are lost as well.
```

File: docs/faq/lukso-standard-proposal.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+However, with smart contract-based accounts like UniversalProfiles, a more flexible and secure solution is available: Social Recovery. With Social Recovery, users can set up a recovery process that fits their needs. For instance, recovery could require permissions from pre-set trusted individuals (often called "guardians") or the provision of a secret phrase known only to the user, or even a combination of these methods.
```
Comment (CJ42) [inline docs/faq/lukso-standard-proposal.md:31]:
```suggestion
However, with smart contract-based accounts like Universal Profiles, a more flexible and secure solution is available: Social Recovery. With Social Recovery, users can set up a recovery process that fits their needs. For instance, recovery could require permissions from pre-set trusted individuals (often called "guardians") or the provision of a secret phrase known only to the user, or even a combination of these methods.
```

File: docs/faq/lukso-standard-proposal.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+This process enables users to regain access to their accounts in a secure way, even if they lose the private key controlling the account, making the blockchain experience much more user-friendly and less risky.
```
Comment (CJ42) [inline docs/faq/lukso-standard-proposal.md:33]:
```suggestion
This process enables users to regain access to their accounts securely, even if they lose the private key controlling the account, making the blockchain experience much more user-friendly and less risky.
```

File: docs/faq/universal-profile.md [hunk 8]
```diff
@@ -27,3 +67,11 @@ LUKSO will migrate to mainnet all the Universal Profiles created on:
 The migration includes all the NFTs issued by [LUKSO](https://lukso.network/) and [The Dematerialised](https://thedematerialised.com).
 
 If your app deploys its contracts and Universal Profiles, you will be responsible for migrating them to the mainnet.
+
+## How is lens protocol different than UniversalProfile?
```
Comment (CJ42) [inline docs/faq/universal-profile.md:71]:
```suggestion
## How is lens protocol different than Universal Profile?
```

**Review by CJ42 — APPROVED** [review]
lgtm. Added few review comments

**Review by b00ste — APPROVED** [review]

**Pull request merged**